ceph.spec.in

```diff
@@ -89,7 +89,7 @@
 %define __python %{__python3}
 %endif
 # unify libexec for all targets
-%global _libexecdir %{_exec_prefix}/lib
+%global _libexecdir %{_exec_prefix}/libexec
```

Comment:
probably we could just drop this line? see 

- http://ftp.rpm.org/api/4.4.2.2/config_macros.html
- https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/#_macros_for_paths_set_and_used_by_build_systems

Reply:
```
[ 5577s] ceph-base-16.0.0-2292.g236c0b7.x86_64.rpm: directories not owned by a package:
[ 5577s]  - /usr/libexec
[ 5577s] ceph-osd-16.0.0-2292.g236c0b7.x86_64.rpm: directories not owned by a package:
[ 5577s]  - /usr/libexec
[ 5578s] rbd-nbd-16.0.0-2292.g236c0b7.x86_64.rpm: directories not owned by a package:
[ 5578s]  - /usr/libexec
```

https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=leap15,DIST=leap15,MACHINE_SIZE=gigantic/41529//consoleFull

it actually breaks the build on openSUSE_Leap_15.2 .